.github/workflows/run_admission_test.yml

```diff
@@ -9,15 +9,6 @@ jobs:
       group: ${{ github.workflow }}-${{ github.ref }}
       cancel-in-progress: true
     steps:
-      - name: Checkout noobaa-core
-        uses: actions/checkout@v3
-        with:
-          repository: 'noobaa/noobaa-core'
-          path: "noobaa-core"
-          # Freeze the version of core
-          # to avoid a failed run due to code changes in the core repo.
-          # Need to update the commit once in a while
-          ref: c97e4ddd2e5fcc110a3242c224eeea97d228c6eb
```

Comment:
@baum, Why not simply update the commit?

Reply:
I agree with removing it. I think it's better to maintain one common place (currently the default image in `options.go`) to determine the test image.